hxrsnd/macromotor.py

```diff
@@ -524,7 +525,10 @@ class DelayMacro(CalibMotor, DelayTowerMacro):
     #                        "t4: {1:.3f}ps".format(t1_delay, t4_delay))
     #     return is_aligned
 
-    calib_detector = Cmp(PCDSAreaDetector, 'XCS:USR:O1000:01:', add_prefix=[])
+    if os.system('caget XCS:USR:O1000:01:') != 256:
```

Comment:
Note that this PV will never exist because it doesn't match anything in the (now offline) IOC exactly

Even if the PV may exist, this creates a 1s timeout on import of macromotor.py when it does not

There's probably some alternate formulation that avoids a timeout

Reply:
Sanghoon said XCS _might_ want to install the area detector again in the future, in which case I was assuming we would use the same IOC. But maybe for now it is better to just remove all those lines and add the area detector again when it is needed?

Reply:
Practically speaking, the quickest/simplest fix is to just remove it as you say.
Possibly we can brainstorm a fix that satisfies both cases without having side effects if we spend some time on it.